native/.idea/codeStyles/Project.xml

```diff
@@ -13,6 +13,24 @@
       <option name="SPACES_WITHIN_OBJECT_LITERAL_BRACES" value="true" />
       <option name="SPACES_WITHIN_IMPORTS" value="true" />
     </JSCodeStyleSettings>
+    <JetCodeStyleSettings>
```

Comment:
This shouldn't be added here as there is no code in the integreat-app-native intellij project, only in the integreat-app-android project.

Reply:
Yeah, you're right. Such unintentional changes are really annoying, I don't even know what caused them...